config/puma.rb

```diff
@@ -0,0 +1,15 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
+threads_count = Integer(ENV['MAX_THREADS'] || 5)
```

Comment:
Are these the correct defaults? I don't think it makes sense to run `boarding` on more than one dyno.